mobile/lib/modules/album/views/album_viewer_page.dart

```diff
@@ -91,6 +91,8 @@ class AlbumViewerPage extends HookConsumerWidget {
             .addAdditionalUserToAlbum(sharedUserIds, album);
 
         if (isSuccess) {
+          // TODO - update state to reflect changes here
```

Comment:
@zoodyy Do you know why even though we invalidated the provider, the state doesn't update here?

Reply:
Yeah, figured it out and added a fix to this branch / PR:

`addAdditionalUserToAlbum` in `AlbumService` did not update the `shared` property of the album - only the shared user list. However, the UI won't show the shared users at all if `shared == false`. This now works for me.

While debugging I found a related server issue: The server does not update the `updatedAt` value of an album if a shared user is added or removed. This still needs to be fixed.

Reply:
Same issue with adding/removing assets to/from an album. server does not update `updatedAt` for the album

Reply:
fixed the server not updating `updatedAt` with #1977